app/_src/guides/otel-metrics.md

```diff
@@ -18,9 +18,6 @@ lots of possibilities of processing and exporting metrics to multiple ecosystems
 {% endif_version %}
 
 {% if_version gte:2.11.x %}
-- Completed [quickstart](/docs/{{ page.release }}/quickstart/kubernetes-demo-kv/) to set up a zone control plane with demo application
```

Comment:
this should stay, but we should update link

Reply:
we have a link above, this should've been completely removed, right?